### DISCUSSION TITLE
installing umap in Pyodide

### DISCUSSION THREAD

**Discussion by cornhundred** [lmcinnes/umap#736] [Q&A]
Hi, I'm trying to install umap-learn in Pyodide but I'm running into an issue that is related to numba support. Is there a version of umap-learn that I can use that does not have numba support or some other way to try to install in Pyodide?

https://github.com/pyodide/pyodide/issues/1739

See example installation here
https://observablehq.com/@cornhundred/pyodide-umap

**Comment by lmcinnes** [comment]
Right now umap is pretty heavily tied to numba. You could try the Julia implementation, or look at UWOT in R that has mostly C++ code.
